.github/PULL_REQUEST_TEMPLAE.md

```diff
@@ -0,0 +1,11 @@
+| Q             | A
+| ------------- | ---
+| Branch?       | "master" for new features / 1.0, 1.1, 1.2, 1.3, 1.4 for fixes
+| Bug fix?      | yes/no
+| New feature?  | yes/no
+| BC breaks?    | yes/no
+| Deprecations? | yes/no
+| Tests pass?   | yes/no
```

Comment:
indeed, i always find that weird with symfony PRs. we could simply remove this line imho.